hooks/terraform-validate.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# OSX GUI apps do not pick up environment variables the same way as Terminal apps and there are no easy solutions,
+# especially as Apple changes the GUI app behavior every release (see https://stackoverflow.com/q/135688/483528). As a
+# workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
+export PATH=$PATH:/usr/local/bin
+
+for dir in $(echo "$@" | tr -s " " "\n" | sort -u | xargs -n1 dirname | uniq); do
```

Comment:
You don't need the `tr` if you are feeding through xargs:
```suggestion
for dir in $(echo "$@" | xargs -n1 dirname | sort -u | uniq); do
```